library/src/main/java/net/yslibrary/licenseadapter/LicenseEntry.java

```diff
@@ -2,15 +2,9 @@
 
 import android.os.Parcelable;
 
-/**
- * Created by yshrsmz on 2016/04/18.
- */
 public interface LicenseEntry extends Parcelable {
-
   void load();
 
-  boolean isLoaded();
```

Comment:
@yshrsmz I didn't realize this was public api and I don't see any reason to deprecate it; some people might be using this. Sorry about that!